Client/Frontend/Browser/OpenWithSettingsViewController.swift

```diff
@@ -66,17 +66,17 @@ class OpenWithSettingsViewController: UITableViewController {
 
     func updateCurrentChoice() {
         var previousChoiceAvailable: Bool = false
-        if let prefMailtoScheme = self.prefs.stringForKey("MailToOption") {
+        if let prefMailtoScheme = self.prefs.stringForKey(PrefsKeys.KeyMailToOption) {
             mailProviderSource.forEach({ (name, scheme, enabled) in
                 if scheme == prefMailtoScheme {
                     previousChoiceAvailable = enabled
                 }
             })
         }
         if !previousChoiceAvailable {
-            self.prefs.setString(mailProviderSource[0].scheme, forKey: "MailToOption")
+            self.prefs.setString(mailProviderSource[0].scheme, forKey: PrefsKeys.KeyMailToOption)
         }
-        self.currentChoice = self.prefs.stringForKey("MailToOption")!
+        self.currentChoice = self.prefs.stringForKey(PrefsKeys.KeyMailToOption)!
```

Comment:
What happens if this pref is not set? Does that mean we crash? Is there a possibility that it is not set? If so, then this should be in a `if let`.